Automattic/wp-calypso
# Jetpack: update Inter font in plans page

## Proposed Changes

* Updates the Inter fonts that were introduced in https://github.com/Automattic/wp-calypso/pull/58671/
* It uses the font Inter Variable, which supports continuous weights of 100 to 900, instead of requesting multiple, discrete weights (Regular, Medium, Semibold, Bold).
* This represents a total savings of ~10% in size, and only 2 HTTP requests (vs 5 before).

Here's the analog diff for jetpack.com: D114865-code

## Testing Instructions

* Fire up this PR.
* Visit the Jetpack pricing page.
* Make sure the variable fonts are being used (e.g.: `Inter-*.var.*`)
